### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by bartek2017** [HermesRepos/android_device_xiaomi_hermes#1]
Hello, I've been using this rom for few days and I have found some bugs.
-watching films in Chrome etc causes green screen flickering
-wifi sometimes (every few hours) disconnects
-there is no root in developer options, only adb or turned off

But generally this Rom is much better than miui, it's fast and I think it manages ram better.

**Comment by Dinolek** [comment]
Hi, are you using latest version of rom (v0.2), it should fix wifi disconnects?

**Comment by bartek2017** [comment]
Yes, but maybe it's my wifi module problem because on miui my wifi has stuck sometimes

**Comment by ChrisjanB** [comment]
Hi 
Installed ROM - Pixel Luncher crashing from time to time.  I'm using Nova Launcher though.
Battery life much better the MIUI .
Keep up the good work!

**Comment by Dinolek** [comment]
"Pixel Luncher crashing from time to time", can you give me log from it(logcat)?

**Comment by ChrisjanB** [comment]
@Dinolek trying to figure out how to get the logcat.

**Comment by ChrisjanB** [comment]
[bugreport-2018-06-27-17-09-23.zip](https://github.com/HermesRepos/android_device_xiaomi_hermes/files/2141952/bugreport-2018-06-27-17-09-23.zip)

**Comment by avcoder0110** [comment]
Sorry for jumped in and my bad English
Many thanks for your work, Dinolek. I'm installed this Rom and it look good.
But could you pls advice how can I have "double tap to wake up screen".
And second, Music player is not working on my phone. It doesn't play any songs, just next next.. from start to end of playlist. Kindly tell me how to fix it.

**Comment by Dinolek** [comment]
About songs, it is codec problem.
Double tap can't be added, because I'm using miui kernel, which don't have this option.

**Comment by AVVincent** [comment]
@Dinolek thank you for your advice.
About songs, i used FLAC files maybe it's problem. So i'll try with MP3.
I want to use double tap because my phone's hard key does not work well. But I fixed it by another way.

**Comment by ChrisjanB** [comment]
"Pixel Luncher crashing from time to time", can you give me log from it(logcat)?
Hopefully the attached is what you need:
[alogcat.2018-06-28-18-40-45+0200.txt](https://github.com/HermesRepos/android_device_xiaomi_hermes/files/2146176/alogcat.2018-06-28-18-40-45.0200.txt)

**Comment by ChrisjanB** [comment]
E/AndroidRuntime( 5248): Process: com.google.android.apps.nexuslauncher, PID: 5248
E/AndroidRuntime( 5248): at com.google.android.apps.nexuslauncher.search.AppSearchProvider.onCreate(SourceFile:89)
E/AndroidRuntime( 6078): Process: com.google.android.apps.nexuslauncher, PID: 6078
E/AndroidRuntime( 6078): at com.google.android.apps.nexuslauncher.search.AppSearchProvider.onCreate(SourceFile:89)
W/BroadcastQueue(  541): Unable to launch app com.google.android.apps.nexuslauncher/10035 for broadcast Intent { act=com.google.android.apps.nexuslauncher.UPDATE_SMARTSPACE flg=0x10000010 pkg=com.google.android.apps.nexuslauncher (has extras) }: process is bad
W/BroadcastQueue(  541): Unable to launch app com.google.android.apps.nexuslauncher/10035 for broadcast Intent { act=com.google.android.apps.nexuslauncher.UPDATE_SMARTSPACE flg=0x10000010 pkg=com.google.android.apps.nexuslauncher (has extras) }: process is bad
W/BroadcastQueue(  541): Unable to launch app com.google.android.apps.nexuslauncher/10035 for broadcast Intent { act=com.google.android.apps.nexuslauncher.UPDATE_SMARTSPACE flg=0x10000010 pkg=com.google.android.apps.nexuslauncher (has extras) }: process is bad
I/ActivityManager(  541): Force stopping com.google.android.apps.nexuslauncher appid=10035 user=0: clear data
W/BroadcastQueue(  541): Unable to launch app com.google.android.apps.nexuslauncher/10035 for broadcast Intent { act=com.google.android.apps.nexuslauncher.UPDATE_SMARTSPACE flg=0x10000010 pkg=com.google.android.apps.nexuslauncher (has extras) }: process is bad
W/BroadcastQueue(  541): Unable to launch app com.google.android.apps.nexuslauncher/10035 for broadcast Intent { act=android.intent.action.PACKAGE_ADDED dat=package:com.google.android.music flg=0x4000010 (has extras) }: process is bad
D/PerformBackupTask(  541): starting key/value backup of BackupRequest{pkg=com.google.android.apps.nexuslauncher}
I/PerformBackupTask(  541): Package com.google.android.apps.nexuslauncher requests full-data rather than key/value; skipping
W/BroadcastQueue(  541): Unable to launch app com.google.android.apps.nexuslauncher/10035 for broadcast Intent { act=com.google.android.apps.nexuslauncher.UPDATE_SMARTSPACE flg=0x10000010 pkg=com.google.android.apps.nexuslauncher (has extras) }: process is bad
W/BroadcastQueue(  541): Unable to launch app com.google.android.apps.nexuslauncher/10035 for broadcast Intent { act=android.intent.action.PACKAGE_ADDED dat=package:rs.pedjaapps.alogcatroot.app flg=0x4000010 (has extras) }: process is bad
W/BroadcastQueue(  541): Unable to launch app com.google.android.apps.nexuslauncher/10035 for broadcast Intent { act=com.android.launcher.action.INSTALL_SHORTCUT flg=0x10 (has extras) }: process is bad
